tests/main/i18n/task.yaml

```diff
@@ -1,5 +1,9 @@
 summary: Test that i18n works
 
+details: |
+  This test cheks the translations do not causes crashes nor warnings.
+  Checks that there are no panics when lancuage packs are installed.
```

Comment:
lancuage => language

Perhaps one line is sufficient? Proposal:
"Checks that there are no translation related panics with language packs installed."

Reply:
done